typescript.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "commitMessageAction": "update",
+  "dependencyDashboard": true,
+  "dependencyDashboardAutoclose": true,
+  "enabledManagers": ["npm", "github-actions", "dockerfile"],
+  "extends": [
+    ":approveMajorUpdates",
+    ":automergeAll",
+    ":combinePatchMinorReleases",
+    ":prConcurrentLimit10",
+    ":prHourlyLimit4",
+    ":prNotPending",
+    "group:nodeJs",
+    "npm:unpublishSafe",
+    "workarounds:typesNodeVersioning"
+  ],
+  "internalChecksAsSuccess": true,
+  "labels": ["dependencies"],
+  "packageRules": [
+    {
+      "commitMessagePrefix": "chore(deps-dev): ",
```

Comment:
don't we also need to configure `chore(deps): ` prefix for production dependencies? or is it the default one?

Reply:
Indeed, default one. But we might as well explicitly define it.

Reply:
Done in fa50655.